Lesson23-aysel/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title>Accordion</title>
+</head>
+<body>
+    <div class="accordion-container">
+        <div class="accordion">
+            <div class="accordion-body">
+                <p>Ready to watch?</p>
+            </div>
+            <div class="accordion-header">
+                <img src="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/5f321b49-5fa7-407b-aaa0-cca6992d3d7c/d5s04qj-d26a072a-3294-4da7-8ab9-a1be78141275.gif?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7InBhdGgiOiJcL2ZcLzVmMzIxYjQ5LTVmYTctNDA3Yi1hYWEwLWNjYTY5OTJkM2Q3Y1wvZDVzMDRxai1kMjZhMDcyYS0zMjk0LTRkYTctOGFiOS1hMWJlNzgxNDEyNzUuZ2lmIn1dXSwiYXVkIjpbInVybjpzZXJ2aWNlOmZpbGUuZG93bmxvYWQiXX0.cBMxrJbp5Q5ghUGAev4ovPgb-Q2evrJs-XqbSrdUeB4" />
+            </div>
+        </div>
+    </div>
+    
+    <div class="container">
+        <div class="img">
+            <img src="images/img1.jpg" alt="">
```

Comment:
Never omit alt text! If you struggle to write it, just ask ChatGPT to 'Create a short alt text to use for HTMl image element' and upload the image that you want to create the text for